cloudflare/resource_cloudflare_zone_cache_variants.go

```diff
@@ -0,0 +1,151 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCloudflareZoneCacheVariants() *schema.Resource {
+	return &schema.Resource{
+		Schema: resourceCloudflareZoneCacheVariantsSchema(),
+		Create: resourceCloudflareZoneCacheVariantsCreate,
```

Comment:
given create and update are essentially the same funcitonality, you could point `Create` and `Update` here to the same method.

Reply:
:+1: I merged both function